src/styles/components/chat/_chat-item-tree-view.scss

```diff
@@ -95,6 +96,18 @@
             min-height: auto;
             height: auto;
             margin: 0;
+            &.mynah-chat-item-tree-view-not-clickable {
+                cursor: default;
+                &:active,
+                &:focus-visible,
+                &:hover {
+                    filter: brightness(0.95);
+                    &:after {
```

Comment:
What does this after within the hover do exactly?

Reply:
That is the flashing cover item, it's main definition is at above. This is just for the animation (which comes over the button) when you hover.